conda_forge_webservices/commands.py

```diff
@@ -643,19 +643,23 @@ def add_user(repo, user):
 
     recipe_path = os.path.join(repo.working_dir, "recipe", "meta.yaml")
     co_path = os.path.join(repo.working_dir, ".github", "CODEOWNERS")
+    yaml = YAML()
```

Comment:
```suggestion
    yaml = YAML(typ="safe")
```

We should be careful since the recipe is from users.

Reply:
Sounds reasonable

We do have a bunch of places where this is not set now. Maybe it's worth filing an issue to fix those up too (not sure if those will be as simple without looking more deeply)?

Reply:
https://github.com/conda-forge/conda-forge-webservices/issues/600